.github/workflows/build.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         id: findcomment
         with:
           issue-number: ${{ github.event.pull_request.number }}
-          comment-author: 'github-actions[bot]'
+          comment-author: "github-actions[bot]"
```

Comment:
No need to change this, but quotes are not needed here. Neither double nor single ones. You need quotes in YAML when you have a string like `"[1,2,3]"` that might represent a JSON object (YAML is a superset of JSON, so every JSON string is also a YAML string).